src/home/marketing/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+
+import { Title } from 'home/marketing/title'
+import { LeftContent } from 'home/marketing/left-side'
+import { RightContent } from 'home/marketing/right-side'
+
+export const Marketing = () => <Grid item xs={10} s={10} md={10} lg={10}>
```

Comment:
These components aren't really `<LeftContent/>` and `<RightContent/>` since they're actually one below the other :)))

Reply:
Done mate :))